sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpInterceptor.kt

```diff
@@ -59,10 +59,17 @@ class SentryOkHttpInterceptor(
             request.body?.contentLength().ifHasValidLength {
                 breadcrumb.setData("request_body_size", it)
             }
-            response?.body?.contentLength().ifHasValidLength {
-                breadcrumb.setData("response_body_size", it)
+
+            val hintsMap = mutableMapOf<String, Any?>("request" to request)
+            response?.let {
+                it.body?.contentLength().ifHasValidLength { responseBodySize ->
+                    breadcrumb.setData("response_body_size", responseBodySize)
+                }
+
+                hintsMap["response"] = it
```

Comment:
```suggestion
                hintsMap["OkHttp:Response"] = it
```

Reply:
Similar to https://github.com/getsentry/sentry-java/pull/1929#discussion_r817034293